tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1350,6 +1350,193 @@ Feature: check share activity
       }
       """
 
+  @issue-10011 @issue-10228
+  Scenario: check share update activities of a folder from a project space
```

Comment:
is there any specific reason to add this scenario in the middle of feature file? If not from next time, please note some file has expected failure scenario, so add scenario at the last

Reply:
Sure, thanks for the suggestion :+1: 
The reason here was to keep similar scenarios together(updating link share and share).

Reply:
ok but this file i dedicated for share activities

Reply:
Yeah, makes sense.
Will keep it in mind next time onwards :+1: